src/postoffice.cc

```diff
@@ -10,12 +10,18 @@
 
 namespace ps {
 Postoffice::Postoffice() {
-  van_ = Van::Create("zmq");
   env_ref_ = Environment::_GetSharedRef();
 }
 
 void Postoffice::InitEnvironment() {
   const char* val = NULL;
+  int enable_rdma = GetEnv("DMLC_ENABLE_RDMA", 0);
```

Comment:
Shall we change the env var to have str value? maybe DMLC_PS_VAN_TYPE = 'tcp', 'libverbs', 'libfabric'? Feel free to suggest better names ..

Reply:
Done. I chose `zmq` over `tcp` as there could be other custom van types that use TCP. I also omitted the prefix `lib` for `libzmq`, `libibverbs`, `libfabric` for brevity.